modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java

```diff
@@ -5933,37 +5935,65 @@ public List<FragmentEntryLinkModel> newObjectFieldsFragmentEntryLinkModels(
 			_getFragmentComponentInputStream("heading", "html"));
 		String paragraphRenderNamespace = StringUtil.randomId();
 
-		for (ObjectFieldModel objectFieldModel : objectFieldModels) {
-			if (objectFieldModel.isSystem()) {
+		for (LayoutModel layoutModel : layoutModels) {
```

Comment:
@slnn Is there any possibility that the `layoutModels` contains more than 2 items?

Reply:
No, it will always contain 2 items

Reply:
I got it,  the layoutModels is an ArrayList, and based on the add order, it is easy to know which one is the hidden layoutModel and which one is the public layoutModel, I will simplify the logic here as no need to use a loop here.